packages/next/shared/lib/dynamic.tsx

```diff
@@ -28,9 +28,13 @@ export type LoadableBaseOptions<P = {}> = LoadableGeneratedOptions & {
     retry?: () => void
     timedOut?: boolean
   }) => JSX.Element | null
-  loader?: Loader<P> | LoaderMap
+  loader?:
+    | Loader<P>
+    | LoaderMap
+    | React.LazyExoticComponent<React.ComponentType<P>>
   loadableGenerated?: LoadableGeneratedOptions
   ssr?: boolean
+  suspense?: boolean
```

Comment:
Since these are public (I think?), would it make sense to make this a union type so that trying to use `suspense` with other options is a type error?